src/components/MoneyTransactionList/MoneyTransactionList.js

```diff
@@ -1,21 +1,25 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import { TableRow } from '../TableRow/TableRow'
 
-export const MoneyTransactionList = ({ moneyTransactions, users, ownId, updateDocument }) => {
-  function findNameById (id) {
+const DEFAULT_USERS = []
+const DEFAULT_TRANSACTIONS = []
```

Comment:
You could define a generic `const EMPTY_ARRAY = []` and reuse it in the whole application